### PR TITLE
[ValloxMV] Add Co2 measurement for ValloxMV binding

### PR DESCRIPTION
**Pull request by mikaelgu80**
Bugfix: Co2 measurement was added to the binding for 3.1.0, but there was an erroneus line that resulted in humidity reading being used as Co2. This PR is meant to fix that issue.

### CODE REVIEW

**Comment by bjoernbrings** [comment]
My first two commits are already included in #10324

**Comment by bjoernbrings** [comment]
Thanks for fetching that mistake

File: bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/i18n/valloxmv_de.properties [hunk 1]
```diff
@@ -1,46 +1,48 @@
 # binding
-binding.valloxmv.name = Vallox L�ftungsanlagen
-binding.valloxmv.description = Binding f�r das Online-Interface von Vallox L�ftungsanlagen
+binding.valloxmv.name = Vallox Lüftungsanlagen
+binding.valloxmv.description = Binding für das Online-Interface von Vallox Lüftungsanlagen
```
Comment (fwolter) [inline bundles/org.openhab.binding.valloxmv/src/main/resources/OH-INF/i18n/valloxmv_de.properties:3]: The properties file shall be ISO 8859-1. You made it UTF-8.

**Review by fwolter — CHANGES_REQUESTED** [review]
Can you fix the conflicting files?

**Comment by mikaelgu80** [comment]
I'll try this again.

**Pull request closed without merging**